libvast/src/table_slice.cpp

```diff
@@ -83,6 +124,16 @@ table_slice::table_slice(chunk_ptr&& chunk, enum verify verify) noexcept {
     fbs::GetTableSlice(data));
 }
 
+table_slice::table_slice(const fbs::FlatTableSlice& flat_slice,
+                         const chunk_ptr& chunk, enum verify verify) noexcept
```

Comment:
Please find a better name for `chunk`. It should be clear that it is a larger surrounding buffer.